src/main/java/org/jabref/gui/slr/ManageStudyDefinitionView.java

```diff
@@ -0,0 +1,326 @@
+        addAuthorButton.setTooltip(new Tooltip(Localization.lang("Add")));
+        addResearchQuestionButton.setTooltip(new Tooltip(Localization.lang("Add")));
+        addQueryButton.setTooltip(new Tooltip(Localization.lang("Add")));
+        addDatabaseButton.setTooltip(new Tooltip(Localization.lang("Add")));
+        addQueryButton.setTooltip(new Tooltip(Localization.lang("Add")));
+        helpIcon.setTooltip(new Tooltip(new StringJoiner("\n")
+                .add(Localization.lang("Query terms are separated by spaces."))
+                .add(Localization.lang("All query terms are joined using the logical AND, and OR operators") + ".")
+                .add(Localization.lang("If the sequence of terms is relevant wrap them in double quotes") + "(\").")
+                .add(Localization.lang("An example:") + " rain AND (clouds OR drops) AND \"precipitation distribution\"")
+                .toString()));
+    }
+
+    private void setKeyPressListenersForInputFields() {
+        addAuthor.setOnKeyPressed(event -> {
+            if (event.getCode() == KeyCode.ENTER) {
+                addAuthor();
+            }
+        });
+        addResearchQuestion.setOnKeyPressed(event -> {
+            if (event.getCode() == KeyCode.ENTER) {
+                addResearchQuestion();
+            }
+        });
+        addQuery.setOnKeyPressed(event -> {
+            if (event.getCode() == KeyCode.ENTER) {
+                addQuery();
+            }
+        });
+        databaseSelectorComboBox.setOnKeyPressed(event -> {
+            if (event.getCode() == KeyCode.ENTER) {
+                addDatabase();
+            }
+        });
+    }
+
+    private void setDataForListViews() {
+        authorListView.setItems(viewModel.getAuthors());
+        questionListView.setItems(viewModel.getResearchQuestions());
+        queryListView.setItems(viewModel.getQueries());
+
+        databaseTableView.setItems(viewModel.getDatabases());
+
+        databaseSelectorComboBox.setItems(viewModel.getNonSelectedDatabases());
+    }
+
+    /**
+     * Configures which cells are used for the different List-/TableViews
+     */
+    private void setCellFactories() {
+        authorListView.setCellFactory(param -> new StringCellWithDelete());
+        questionListView.setCellFactory(param -> new StringCellWithDelete());
+        queryListView.setCellFactory(param -> new StringCellWithDelete());
+
+        databaseSelectorComboBox.setCellFactory(param -> new StudyDatabaseComboBoxCell());
+        databaseSelectorComboBox.setButtonCell(new StudyDatabaseComboBoxCell());
+        databaseTableEntry.setCellValueFactory(param -> new ReadOnlyStringWrapper(param.getValue().getName()));
+        enabledTableEntry.setCellValueFactory(param -> new ReadOnlyObjectWrapper<>(param.getValue()));
+        removeTableEntry.setCellValueFactory(param -> new ReadOnlyObjectWrapper<>(param.getValue()));
+
+        enabledTableEntry.setCellFactory(param -> new TableCell<>() {
```

Comment:
Why didn't you reuse the existing ViewModelCellFactory methods? They handle much of the boilerplate code for you.